src/apps/vpn/ui/screens/settings/ViewDNSSettings.qml

```diff
@@ -135,10 +135,11 @@ VPNViewBase {
                     enabled: VPNSettings.dnsProviderFlags === VPNSettings.Custom
                     onEnabledChanged: if(enabled) forceActiveFocus()
 
-                    _placeholderText: VPN.placeholderUserDNS
+                    _placeholderText: VPNl18n.SettingsDnsSettingsInputPlaceholder
                     text: ""
                     Layout.fillWidth: true
                     Layout.topMargin: 12
+                    inputMethodHints: Qt.ImhNoPredictiveText | Qt.ImhFormattedNumbersOnly
```

Comment:
Have you tested what keyboard this gives you on android? I'm curious because even though iOS doesn't have one, I seem to remember Android having a digits + "." and "," keyboard (which you might get from `Qt.ImhDigitsOnly`). If so, would it even be worth it to have different keyboards between iOS and Android?